src/gcinfo/CMakeLists.txt

```diff
@@ -23,5 +23,15 @@ endif(CLR_CMAKE_PLATFORM_UNIX)
 add_subdirectory(lib)
 add_subdirectory(crossgen)
 
+if (CLR_CMAKE_PLATFORM_ARCH_I386)
+    # On x86, build RyuJIT/ARM32 cross-compiling altjit.
+    add_subdirectory(arm_altjit)
```

Comment:
Regardless, this is a separate component and should be named as such. It is confusing to navigate to a directory named "gcinfo" and find subdirectories referencing the JIT. For example, when I first looked at the PR, I thought that you had actually placed the directories underneath the JIT's directory (e.g. `src/jit/arm{,64}_altjit`).